domains/oliik2013.json

```
@@ -17,7 +17,8 @@
               "mx1.improvmx.com",
               "mx2.improvmx.com"
             ],
-            "TXT": "v=spf1 include:spf.improvmx.com ~all"
+            "TXT": "v=spf1 include:spf.improvmx.com ~all",
+            "TXT": "zoho-verification=zb43869934.zmverify.zoho.eu"
```

You can't have 2 TXT records, Merge them into an array
```suggestion
            "TXT": ["v=spf1 include:spf.improvmx.com ~all", "zoho-verification=zb43869934.zmverify.zoho.eu"]
```